snippets/filterFalsy.md

```diff
@@ -0,0 +1,13 @@
+### filterFalsy
+
+Filters out the falsy values in an array.
+
+Use `Array.prototype.filter()` to get an array containing only truthy values.
+
+```js
+const filterFalsy = arr => arr.filter(Boolean)

```

Comment:
Just a little nit (just happened to be around), this line lacks of a semicolon.

Reply:
Nice catch, it's all that standard is style getting into me :D